content/html/concepts/text area/text area.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Text Area'
+Description: 'Text area helps you with writting more than one line instead od using input type of text'
```

Comment:
Since this is meant to be a term entry, can we rewrite the `Description` such that it starts with a present-tense verb such as this?

```suggestion
Description: 'Represents a multiline input form on a webpage.'
```